lingon/src/semantics.rs

```diff
@@ -0,0 +1,108 @@
+//! Stuff used internally for rendering, probably
```

Comment:
We might want a `renderer` submodule, like an actual folder. Don't know how much more is needed in the renderer though.

But I really disliked all the auto-generated clutter from Luminance, so I refactored it, quick and dirty.